packages/lesswrong/components/notifications/NotificationsMenuButton.tsx

```diff
@@ -13,12 +13,18 @@ const styles = (theme: ThemeType): JssStyles => ({
   badge: {
     backgroundColor: 'inherit',
     color: theme.palette.header.text,
-    fontFamily: 'freight-sans-pro, sans-serif',
-    fontSize: "12px",
     fontWeight: 500,
     right: "1px",
     top: "1px",
     pointerEvents: "none",
+    ...(isEAForum
+      ? {
+        fontSize: 10,
+      }
+      : {
+        fontFamily: "freight-sans-pro, sans-serif",
```

Comment:
The hell is this? I was wondering how the heck our old font was even sticking around.